src/coregraphics.cs

```diff
@@ -76,6 +76,14 @@ interface CGPDFInfo {
 		[Mac (10,13)][iOS (11,0)][TV (11,0)][Watch (4,0)]
 		[Internal][Field ("kCGPDFContextAccessPermissions")]
 		IntPtr kCGPDFContextAccessPermissions { get; }
+
+		[Mac (10,16)][iOS (14,0)][TV (14,0)][Watch (7,0)]
+		[Field ("kCGPDFContextCreateLinearizedPDF")]
+		IntPtr kCGPDFContextCreateLinearizedPDF { get; }
+
+		[Mac (10,16)][iOS (14,0)][TV (14,0)][Watch (7,0)]
+		[Field ("kCGPDFContextCreatePDFA")]
+		IntPtr kCGPDFContextCreatePDFA { get; }
```

Comment:
Internal or a better name?

Reply:
same, the better name is in the _old/manual_ strong dictionary